### PR TITLE
Add Connector API interfaces

### PR DESCRIPTION
**Pull request by alextremp**
This PR includes:

* The AdConnectorSourceId interface for connectors
* The AdLoadable interface for connectors supporting positions that can be loaded without being displayed
* The AdViewable interface for connectors to implement the display and refresh position methods

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    "prepublishOnly": "npm run cleanDistFolder && npm run build",
+    "build": "npm run cleanDistFolder && babel src --ignore test --out-dir dist ",
+    "lint": "sui-lint js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/scm-spain/OpenAds-ConnectorApi"
+  },
+  "keywords": [
+    "appnexus",
```
Comment (victuxbb) [inline package.json]: Ein?

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import AdConnectorSourceId from './interface/AdConnectorSourceId'
+import AdLoadable from './interface/AdLoadable'
+import AdViewable from './interface/AdViewable'
+
+export {
+  AdConnectorSourceId,
```
Comment (victuxbb) [inline src/index.js]: After talking about id source, this could be deleted

Comment (alextremp) [reply src/index.js]: Done

File: package.json (hunk 1 above)
Comment (alextremp) [reply package.json]: What ¯\_(ツ)_/¯

Comment (victuxbb) [reply package.json]: This is supposed to be interfaces agnostic from appnexus or ast or whatever connector...

Comment (alextremp) [reply package.json]: You're right :) It's changed

**Review by pablovegau — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,2 +1,16 @@
 # OpenAds-ConnectorAPI
-Complaint interfaces to be implemented to develop an OpenAds connector
+
+Compliant interfaces to be implemented to develop an OpenAds connector.
```
Comment (rafamarquezv) [inline README.md:3]: I would use: 

_Compliant interfaces to be implemented when developing an OpenAds connector._

File: package.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@schibstedspain/openads-connector-api",
+  "version": "1.0.0",
+  "description": "OpenAds Interfaces to be compiant developing a new OpenAds connector",
```
Comment (rafamarquezv) [inline package.json]: Typo:

_"description": "OpenAds Interfaces to be compliant when developing a new OpenAds connector",_

**Review by rafamarquezv — CHANGES_REQUESTED** [review]
Great job! 👏 
Just 1 typo and 1 suggestion... ✍️

File: README.md (hunk 3 above)
Comment (alextremp) [reply README.md:3]: It was already fixed 👍

File: package.json (hunk 4 above)
Comment (alextremp) [reply package.json]: Fixed! thanks 👍

**Review by rafamarquezv — APPROVED** [review]

**Pull request merged**